benchmark/benchmain/main.go

```diff
@@ -326,6 +325,38 @@ func readTimeFromInput(values *[]time.Duration, replace string) {
 	}
 }
 
+func partialPrintString(noneEmptyPos []bool, f bm.Features, shared bool) string {
+	s := ""
+	var prefix, suffix string
+	if shared {
+		suffix = "\n"
+	} else {
+		prefix = "-"
+	}
+	if noneEmptyPos[1] {
+		s += fmt.Sprintf("%slatency_%s%s", prefix, f.Latency.String(), suffix)
```

Comment:
If these hard-to-read strings aren't important to the program, we should output them in a more readable format.

E.g.
```
Network: LAN, Parallel Calls: 1, ReqSz: 1B, RespSz: 1000000B, Compressor
Network: WAN, Parallel Calls: 10, ReqSz: 1000000B, RespSz: 1B, !Compressor
```

Reply:
Done. I move this part to stats.go because later fmt.go can use it to format output.